app/src/main/java/com/example/neighbourproject/HomePageActivity.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.neighbourproject
+
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+
+lateinit var registerButton: TextView
+lateinit var usernameEdit: EditText
+lateinit var passwordEdit: EditText
+
+class HomePageActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_home_page)
+
+        registerButton = findViewById(R.id.registerTextview)
+        usernameEdit = findViewById(R.id.usernameEditText)
+        passwordEdit = findViewById(R.id.passwordEditText)
+
+        emailCheckCorrect()
+
+
+
+
+        val loginButton = findViewById<Button>(R.id.loginButton)
+        loginButton.setOnClickListener{
+            val intent = Intent(this, MainActivity::class.java)
```

Comment:
I'm a bit confused around this intent to MainActivity. Anyhow to get this activity to work it need's to be defined in the manifest